governance/hardhat.config.js

```diff
@@ -67,7 +67,10 @@ const config = {
   networks,
   etherscan,
   solidity: {
-    compilers: [{ version: '0.8.21', settings }],
+    compilers: [
+      { version: '0.7.6', settings }, // required to compile oracle
```

Comment:
Do we want to put that settings in the `contracts` package maybe?

Reply:
The issue here is that we need to compile from the governance folder in order to verify the contract. Another approach will be to upgrade our oracle contract to use `0.8.21` so all contracts use latest version